stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -452,6 +457,7 @@ function(_compile_swift_files
       "${SWIFT_STDLIB_BUILD_TYPE}"
       "${SWIFT_STDLIB_ASSERTIONS}"
       swift_flags
+      DEPLOYMENT_VERSION_OSX ${SWIFTFILE_DEPLOYMENT_VERSION_OSX}
```

Comment:
I'm surprised that any of these changes are needed, given that `add_swift_target_library` is already being used with these `DEPLOYMENT_VERSION_*` options elsewhere. Are you sure these changes here are necessary?

Reply:
`add_swift_target_library` currently only applies these options for C/C++ source files. Here we're making it apply `DEPLOYMENT_VERSION_*` for Swift modules as well.